content/30.conclusions.md

```diff
@@ -14,12 +14,12 @@ Though this effect has been studied with respect to gender, we are not aware of
 
 We acknowledged that our supervised learning approaches are neither error free nor bias free.
 Because wru was trained on the US census to make predictions, many of the missing predictions are on names not observed in the US census.
-Although the underestimation of the proportion of these names could not be compared between the list of authors and honorees, we complemented this race/ethnicity prediction method with an additional name origin analysis.
+Although the underestimation of the proportion of these names could not be quantified in the list of authors and honorees, we complemented this race/ethnicity prediction method with an additional name origin analysis.
 By integrating different methods and preserving uncertainty by analyzing prediction probabilities rather than applying a hard assignment for each prediction, we hope to alleviate these biases and discover insightful findings that correctly reflect the current representation diversity at conferences.
 
 Focusing on an international society and meetings, we measured honor and authorship rates worldwide.
 In this setting, we observe disparities by name groups.
-Because invitation and honor patterns could be driven by biases associated with name groups, geography, or other factors, we cross-referenced name group predictions with author affiliations could help to disentangle the relationship between geographic regions, name groups and invitation probabilities.
+Because invitation and honor patterns could be driven by biases associated with name groups, geography, or other factors, we cross-referenced name group predictions with author affiliations to disentangle the relationship between geographic regions, name groups and invitation probabilities.
```

Comment:
```suggestion
Because invitation and honor patterns could be driven by biases associated with name groups, geography, or other factors, we cross-referenced name group predictions with author affiliations to disentangle the relationship between geographic regions, name groups and invitation probabilities.
We found that disparities persisted even within honorees with a US affiliation.
```